client/src/components/Home/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from 'antd';
+import './style.css';
+
+function Home() {
+  return (
+    <section className='home'>
+      <h1 className='home--heading'>
```

Comment:
i think you forget the logo of the page. 
```suggestion
      <p className='home--desc'>
```

Reply:
I don't need to add the logo since we will have the header component @knanahassouna1